### PR TITLE
Added psuedo-setters for context properties

### PR DESCRIPTION
**Pull request by pseusys**

### CODE REVIEW

**Review by kudep — CHANGES_REQUESTED** [review]
Omg, why did we add old history of commits?

**Comment by pseusys** [comment]
I've started this branch from `merge/telegram` by mistake I think, hut I've left only the changes that belong here.

File: examples/generics/2_buttons.py [hunk 1]
```diff
@@ -139,22 +138,15 @@ class CallbackRequest(NamedTuple):
...
         try:
-            chosen_button = ui.buttons[int(last_request)]
+            chosen_button = ui.buttons[int(ctx.last_request)]
         except (IndexError, ValueError):
             raise ValueError(
                 "Type in the index of the correct option" "to choose from the buttons."
             )
-        ctx.requests[last_index] = CallbackRequest(payload=chosen_button.payload)
-        return
-    ctx.requests[last_index] = last_request
```
Comment (kudep) [inline examples/generics/2_buttons.py:157]: same issue of removing

**Review by kudep — CHANGES_REQUESTED** [review]
- [ ] fix examples

**Review by kudep — APPROVED** [review]

**Pull request merged**